Driver search results

MichaelParadis opened the pull request.

MichaelParadis committed:
Merge branch 'request-elastics-integration2' of github.com:CMPUT301F16T01/Carrier into driver-search-results

MichaelParadis committed:
Added things

MichaelParadis committed:
Merge branch 'request-elastics-integration2' of github.com:CMPUT301F16T01/Carrier into driver-search-results

MichaelParadis committed:
Merge branch 'master' of github.com:CMPUT301F16T01/Carrier into driver-search-results

MichaelParadis committed:
Added cancelling requests to the elastic search

MichaelParadis committed:
Merged master to fix merge conflict

MichaelParadis committed:
Added stuff

MichaelParadis committed:
Merge branch 'master' of github.com:CMPUT301F16T01/Carrier into driver-search-results

MichaelParadis committed:
added stuff

MichaelParadis committed:
Merge branch 'master' of github.com:CMPUT301F16T01/Carrier into driver-search-results

Hreherch commented:
The tests don't build and I can't tell what you worked on by looking at your commits. :(
Also I thought Sarah was doing the searching tests?

Hreherch committed:
Request statuses update according to actions

Need to add checks to ensure everything does not break when something
unexpected happens!

sehoven committed:
Merge branch 'request-stati' of github.com:CMPUT301F16T01/Carrier into request-stati

Hreherch committed:
Using RequestAdapter not DriverRequestAdapter

sehoven committed:
Merge branch 'request-stati' of github.com:CMPUT301F16T01/Carrier into request-stati

Hreherch committed:
Listeners are notified when changing statuses

sehoven committed:
Made currency locale-based, fixed getCenter()

sehoven committed:
Merge pull request #123 from CMPUT301F16T01/request-stati-improve

Request stati improve

MichaelParadis committed:
Merge branch 'master' of github.com:CMPUT301F16T01/Carrier into driver-search-results

MichaelParadis commented:
@Hreherch This was linking the search results to the driver view request activity. The tests we changed to allow them to build but i just pulled master again so it will be the same as what master has.

MichaelParadis committed:
Changed the request adapter for driver to use the requestslist_item
instead of the one that I made before

MichaelParadis committed:
Merge branch 'request-stati' of github.com:CMPUT301F16T01/Carrier into driver-search-results
